core/src/test/java/org/elasticsearch/search/aggregations/bucket/DateHistogramOffsetIT.java

```diff
@@ -58,6 +61,11 @@ private DateTime date(String date) {
     }
 
     @Override
+    protected Collection<Class<? extends Plugin>> nodePlugins() {
+        return Collections.singleton(AssertingLocalTransport.TestPlugin.class);
```

Comment:
if you set the setting you have to register it so you can only set the setting if you ahve the plugin installed